doc/source/reference/algorithms.centrality.rst

```diff
@@ -57,17 +57,15 @@ Eigenvector
    katz_centrality
    katz_centrality_numpy
 
-Communicability
+Subgraph
 ---------------
 .. autosummary::
    :toctree: generated/
 
-   communicability
-   communicability_exp
-   communicability_centrality
-   communicability_centrality_exp
-   communicability_betweenness_centrality
-   estrada_index
+    subgraph_centrality
```

Comment:
*correcting* is maybe the wrong word: I removed entries that don't fit here any longer. They are now in ` doc/source/reference/algorithms.communicability.rst`.